MDB2/Driver/Manager/pgsql.php

```diff
@@ -345,8 +348,13 @@ public function alterTable($name, $changes, $check)
             case 'rename':
                 break;
             default:
-                return $db->raiseError(MDB2_ERROR_CANNOT_ALTER, null, null,
-                    'change type "'.$change_name.'\" not yet supported', __FUNCTION__);
+                return $db->raiseError(
+                    MDB2_ERROR_CANNOT_ALTER,
+                    null,
+                    null,
+                    'change type "' . $change_name . '" not yet supported',
```

Comment:
You've removed the backslash from this line but not line 411, is there a reason for this?